controllers/customers.js

```diff
@@ -0,0 +1,48 @@
+const models = require('../models')
+const Customer = require('../models/customer')
+
+
+const handleUploadCustomers = customers => {
+  const customerEmailValidate = customers.map(customer => {
+    const { email } = customer
+    // removes whitespaces and commas
+    const invalidChars = /[,\s]+|[,\s]+/g
+
+    const validatedEmail = email.replace(invalidChars, '')
+
+    return {
+      ...customer,
+      'email': validatedEmail,
```

Comment:
No need to add the quotes here when adding a key to an object in javascript `email: validateEmail` should work